provider/sriov/sriov_base.py

```diff
@@ -258,12 +258,11 @@ def get_rom_file(self):
         rom_vendor_device = lspci_stdout[1:-1].replace(':', '') + '.rom'
         rom_file = os.path.join('/usr/share/ipxe', rom_vendor_device)
         if not os.path.exists(rom_file):
-            build_cmd = "git clone https://github.com/ipxe/ipxe.git;\
-                         pushd ipxe/src; make bin/{0}; cp bin/{0} {1}; popd; \
-                         rm -rf ipxe".format(rom_vendor_device, rom_file)
-            process.run(build_cmd, shell=True, verbose=True)
+            # Directly use virtio rom file because it mainly test xml but not pxe function.
+            rom_file = "/usr/share/ipxe/1af41000.rom"
```

Comment:
I think you can use a more direct variable name instead of commenting here.
```suggestion
            virtio_rom_file = "/usr/share/ipxe/1af41000.rom"
            if not os.path.exists(virtio_rom_file):
                self.test.error(f"This test needs a rom file, but neither {virtio_rom_file} or {rom_file} exist!")
```

Reply:
At the end of this function, we nee to `return rom_file` to use in calling function. So virtio_rom_file is not suitable here.

Reply:
yes, of course you can return virtio_rom_file if it exists.

Reply:
Updated.